blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineUpdateRequest.java

```diff
@@ -55,6 +55,7 @@ public BluePipeline update(BluePipeline pipeline) throws IOException {
                 folder.scheduleBuild(new Cause.UserIdCause());
             }
         }
+        item.save();
```

Comment:
item.save() is not needed, in fact should be removed - extra cost of saving, since folder.getNavigators().replace(gitHubSCMNavigator); does save underneath.